lib/Sema/SemaBounds.cpp

```diff
@@ -2151,6 +2156,62 @@ namespace {
       return true;
     }
 
+    // This function is an extension of EqualExtended. It looks into the provided `Facts`
+    // in order to prove `Base1 + Offset1 <= Offset2 + Offset2`.
```

Comment:
There's typo here: Offset2 is used twice.

Could you add a TODO comment: we are ignoring the possibility of overflow in the addition.